moped-editor/src/views/dashboard/DashboardView.js

```diff
@@ -0,0 +1,245 @@
+import React, { useState } from "react";
+import { useQuery } from "@apollo/client";
+
+// Material
+import {
+  AppBar,
+  Box,
+  Card,
+  CircularProgress,
+  Container,
+  Grid,
+  Tab,
+  Tabs,
+  Typography,
+} from "@material-ui/core";
+import MaterialTable from "@material-table/core";
+import { makeStyles } from "@material-ui/core/styles";
+
+import Page from "src/components/Page";
+
+import RenderFieldLink from "../projects/signalProjectTable/RenderFieldLink";
+import ProjectStatusBadge from "../projects/projectView/ProjectStatusBadge";
+
+import typography from "../../theme/typography";
+
+import {
+  USER_FOLLOWED_PROJECTS_QUERY,
+  USER_PERSONNEL_PROJECTS_QUERY,
+} from "../../queries/dashboard";
+import { STATUS_QUERY } from "../../queries/project";
+
+import { getSessionDatabaseData } from "../../auth/user";
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    flexGrow: 1,
+    backgroundColor: theme.palette.background.paper,
+  },
+  appBar: {
+    backgroundColor: theme.palette.background.paper,
+    color: theme.palette.text.header,
+  },
+  cardWrapper: {
+    marginTop: theme.spacing(3),
+  },
+  selectedTab: {
+    "&.Mui-selected": {
+      color: theme.palette.text.primary,
+    },
+  },
+  indicatorColor: {
+    backgroundColor: theme.palette.primary.light,
+  },
+}));
+
+function a11yProps(index) {
+  return {
+    id: `simple-tab-${index}`,
+    "aria-controls": `simple-tabpanel-${index}`,
+  };
+}
+
+const TABS = [
+  {
+    label: "My projects",
+    query: USER_PERSONNEL_PROJECTS_QUERY,
+  },
+  {
+    label: "Following",
+    query: USER_FOLLOWED_PROJECTS_QUERY,
+  },
+];
+
+const DashboardView = () => {
+  const userSessionData = getSessionDatabaseData();
+  const userId = userSessionData.user_id;
+
+  const classes = useStyles();
+  const typographyStyle = {
+    fontFamily: typography.fontFamily,
+    fontSize: "14px",
+  };
+
+  const [activeTab, setActiveTab] = useState(0);
+
+  const { loading, error, data } = useQuery(TABS[activeTab].query, {
```

Comment:
I think you have may have asked my opinion on this early in the implementation so sorry if I'm contradicting my past self, but I think this Dashboard view would perform more smoothly if we called the combined query for both following and user personnel projects 1 time instead of calling it each time the active Tab switches. 

Basically, on page load, we should know any and all projects we'll need and we should let React state handle the toggling of that data instead of putting that work on the DB. I'm going to make a new issue for this enhancement. 

Attaching a video demo of what I'm describing. Note 1) the flash of the empty table on the re-render caused by the active tab toggle and 2) the multiple unnecessary GraphQL calls.

https://user-images.githubusercontent.com/5697474/171481159-833d37db-cb02-413f-a406-073729c61473.mp4

Reply:
https://github.com/cityofaustin/atd-data-tech/issues/9349